packages/rich-text/src/test/insert-object.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+
+import { insertObject } from '../insert-object';
+import { getSparseArrayLength } from './helpers';
+
+describe( 'insert', () => {
+	const obj = { type: 'obj' };
+	const em = { type: 'em' };
+
+	const OBJECT_REPLACEMENT_CHARACTER = '\ufffc';
```

Comment:
There's a special characters file with these constants that you can import.

Reply:
Oh yeah, I noticed that. I was working off an older version that didn't have that implemented yet. I assume updating the branch would require me to take care of the merge conflict myself?

Reply:
I usually do the following on my branch:
```bash
git fetch
git rebase origin/master
```
you fix conflicts and:
```bash
git add .
git rebase --continue
```